### PR TITLE
Perf: reduce bundle size and nextjs 12 config

### PR DESCRIPTION
**Pull request by sambacha**
Thanks for providing this template, some minor tweaks

Cheers

### CODE REVIEW

File: .vercelignore [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# -*- mode: gitignore; -*-
```
Comment (mirshko) [inline .vercelignore:1]: I don't think this file is needed, there isn't any of these files in the next source output tab in Vercel.

Comment (mirshko) [reply .vercelignore:1]: Re: The tests, they should not be colocated with the api routes, or if so they should be prefixed with `_` so they are treated as helper files not source routes.

**Review by mirshko — COMMENTED** [review]
This is a little too opinionated for the scope of this project, see my comment on the .vercelignore, the next.config.js config options are bare minimum intentionally.

**Comment by sambacha** [comment]
thanks for feedback, much appreciated

**Pull request closed without merging**